ui/tests/integration/components/sync/secrets/page/overview-test.js

```diff
@@ -30,357 +30,344 @@ module('Integration | Component | sync | Page::Overview', function (hooks) {
     this.server.post('/sys/capabilities-self', allowAllCapabilitiesStub());
     this.version = this.owner.lookup('service:version');
     this.store = this.owner.lookup('service:store');
-    this.version.type = 'enterprise';
-    this.version.features = ['Secrets Sync'];
+    this.flags = this.owner.lookup('service:flags');
 
     syncScenario(this.server);
     syncHandlers(this.server);
 
     this.destinations = await this.store.query('sync/destination', {});
-    this.isActivated = true;
-    this.licenseHasSecretsSync = true;
-    this.isHvdManaged = false;
```

Comment:
it was really hard to tell what the context was for each test below. though the module organization attempted to clarify, it was inconsistent with how/what was set. 

I opted to create the `setup` helper to default everything to false and make it super clear what version logic was used in each test. (it made logic gaps more obvious, so i added tests for those! 🎉 )